lib/util/cache/repository/types.ts

```diff
@@ -1,5 +1,5 @@
 import type { PackageFile } from '../../../modules/manager/types';
-import type { RepoInitConfig } from '../../../workers/repository/init/types';
+import type { RepoInitConfig } from '../../../workers/repository/config/types';
```

Comment:
We should refactor this later, so we don't have references to `workers` from `util`